frontend-ssr/services/ExerciseSubmissions.service.ts

```diff
@@ -0,0 +1,8 @@
+import HttpService from './Http.service';
+
+export default async function getSubmissions(page: number, query: string) {
+  const responseSubmissions = await HttpService.get(`${process.env.ENDPOINT}/exercises/submissions?search=al`)
+    .then((response) => response.json())
+    .catch((err) => alert(err));
```

Comment:
We shouldn't handle the error here but in the react component. Also, no point for async-await or `.then` here.

I think the HttpServic already handles converting to `json` so in this function you should just return the call to the service, no async-await needed.

PS: let's add some types as well, that would be amazing:

`: Promise<Submission[]>`